theconfluenceBOT/new.py

```diff
@@ -24,8 +24,7 @@
 WHITELIST = [
     "Zokalyx",
     "MoscaMye",
-    "theconfluencer",
-    "Thejusko"
```

Comment:
You can still add yourself - but I understand that you cannot change the DEBUG variable. Do you think it's a good idea to implement some sort of keyword that corresponds to debugging? Results would be sent back through DMs, but not posted on the profile. I'm not saying you should add that, I can do it but I just wanted to know your opinion.